markovs_household/data/read_probabilities.py

```diff
@@ -0,0 +1,46 @@
+import os
+import pandas as pd
+import numpy as np
+from markovs_household.config.config import Configuration
+
+
+def calculate_probabilities(df, type):

```

Comment:
Ich hätte hier einen Vorschlag wie man das vielleicht ein wenig anders aufziehen könnte. 
Wenn ich das richtig überschaue berechnest du hier die (Einschalt?)wahrscheinlichkeiten pro Gerät separat - richtig? 
Da wir wenn wir die berechnen wollen grundsätzlich alle Wahrscheinlichkeiten der Geräte berechnen kann man das ganze in einem machen und in eine separate Datenklasse stecken die zum Beispiel `Einschaltwahrscheinlichkeiten` heißen könnte und als Attribute jeweils die einzelnen Geräte trägt und dort die jeweiligen Einschaltwahrscheinlichkeiten des Geräts hinterlegt.